services/hanke-service/src/main/resources/db/changelog/changesets/086-change-car-lane-nuisance-enum.sql

```diff
@@ -0,0 +1,134 @@
+--liquibase formatted sql
+--changeset Teemu Hiltunen:086-change-car-lane-nuisance-enum
+--comment: Change 'hankealue.kaistahaitta' enum values and 'tormaystarkastelutulos.kaistapituushaitta' and 'applications.applicationData.areas[n].tyoalueet[m].tormaystarkastelutulos.kaistapituushaitta' index values
+
+-- update hankealue.kaistahaitta enum values
+ALTER TABLE hankealue
+    ADD COLUMN kaistahaitta_temp VARCHAR;
+
+UPDATE hankealue
+    SET kaistahaitta_temp =
+        CASE
+            WHEN kaistahaitta = 'VAHENTAA_KAISTAN_YHDELLA_AJOSUUNNALLA' THEN 'YKSI_KAISTA_VAHENEE'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_KAISTAN_KAHDELLA_AJOSUUNNALLA' THEN 'YKSI_KAISTA_VAHENEE_KAHDELLA_AJOSUUNNALLA'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_USEITA_KAISTOJA_KAHDELLA_AJOSUUNNALLA' THEN 'USEITA_KAISTOJA_VAHENEE_AJOSUUNNILLA'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_USEITA_KAISTOJA_LIITTYMIEN_ERI_SUUNNILLA' THEN 'USEITA_AJOSUUNTIA_POISTUU_KAYTOSTA'
+        END;
```

Comment:
This returns null if the old value is `EI_VAIKUTA`. There should be a ELSE clause like
```
            ELSE kaistahaitta
```

Reply:
Added the ELSE clause